zScanner/SupportFiles/Extensions/String.swift

```diff
@@ -41,4 +41,12 @@ extension String {
         return String(self[start ..< end])
     }
     
+    /* if the personal identification number (PIN - YYMMDD/___?) a has length equal to 9 so year must be 53 and less othewise length has to be 10 */
```

Comment:
Reformat the comment. Don't describe the algorithm but describe the reason behind it. E.g..: The numbers before the year 1954 had only 9 digits, but starting 1954, they changed it to 10